source/quickstart.txt

```diff
@@ -37,6 +37,10 @@ the rest of the {+c2c-product-name+} documentation.
   meet the minimum patch :ref:`version requirements
   <c2c-server-version-compatibility>`.
 
+- The source cluster remains active during sync. ``mongosync``
+  syncs new writes on the source cluster for the duration of the
+  migration.
```

Comment:
```suggestion
- The source cluster can remain active during a migration until commit because ``mongosync``
  syncs writes on the source cluster for the duration of the
  migration.
```

Reply:
Something like this. 
- The source cluster can remain active, doesn't necessarily need to
- Writes need to be stopped pre commit
- Took out the word new because it syncs existing data too

Reply:
Fixed